### ISSUE TITLE
Scrolling Issue with ScrollMode.OffScreen

### ISSUE THREAD

**Issue by Danielku15** [CoderLine/alphaTab#1766]
### Discussed in https://github.com/CoderLine/alphaTab/discussions/1754

<div type='discussions-op-text'>

<sup>Originally posted by **goodgame365** November 14, 2024</sup>
I noticed that when public scrollMode: ScrollMode = ScrollMode.OffScreen;, if the cursor reaches the bottom, scrolling is no longer needed. However, when repeatedly replying, if the starting point exceeds the display range, the screen does not jump to that location.

On the other hand, if public scrollMode: ScrollMode = ScrollMode.Continuous;, this issue does not occur.

I’m not sure if this is a bug.</div>

In short: with ScrollMode.OffScreen, the scrolling "up" to repeat starts seem to be broken.
